packages/mascot/tests/e2e/registration.it.test.ts

```
@@ -0,0 +1,264 @@
+    const harness = window.mascotHarness
+    const service = harness.mascot.createMascot({state: 'rest', working: false, follow: false})
+    const first = service.connect()
+    const second = service.connect()
+    const names: ConnectSlot[] = [
+      'getRootProps',
+      'getHeadProps',
+      'getEyesProps',
+      'getAntennaProps',
+      'getEffectHostProps',
+    ]
+    const sameGetterTwice = names.every((name) => first[name]().ref === first[name]().ref)
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)